backend/typescript/services/implementations/userService.ts

```diff
@@ -253,7 +253,7 @@ class UserService implements IUserService {
                   lastName: oldUser.lastName,
                   role: oldUser.role,
                   active: oldUser.active,
-                  camps: (oldUser as CampLeader).camps,
+                  camps: (oldUser as CampLeader).campSessions,
```

Comment:
think we need to change all the `camps` in this file to `campSessions`  - do you mind making that change for now? @amanda-guo should be updating some of this code in her PR as well